packages/components/src/components/wallet-cfd-success-dialog/wallet-cfd-card.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import classNames from 'classnames';
+import Text from '../text';
+import { isMobile } from '@deriv/shared';
+import { AppLinkedWithWalletIcon } from '../app-linked-with-wallet-icon';
+import { TWalletCFDCard } from './wallet-cfd-success-dialog';
+import './wallet-cfd-card.scss';
+
+const WalletCFDCard = ({ wallet }: { wallet: TWalletCFDCard }) => {
+    const {
+        account_title,
+        app_icon,
+        balance,
+        currency,
+        currency_title,
+        gradient_header_class,
+        icon,
+        is_demo,
+        wallet_label,
+    } = wallet;
+
+    return (
+        <div className='wallet-cfd-card'>
+            <div className='wallet-cfd-card__bg'>
+                <div className='wallet-cfd-card__shine' />
+                <div className='wallet-cfd-card__icon'>
+                    <AppLinkedWithWalletIcon
+                        size={isMobile() ? 'small' : 'medium'}
+                        app_icon={app_icon}
+                        gradient_class={gradient_header_class}
+                        hide_watermark
+                        type={is_demo ? 'demo' : 'fiat'}
+                        wallet_icon={icon}
+                    />
+                </div>
+                <div
```

Comment:
Try to use `Badge` component inside of the tsx file with `rounded_corners={2}` instead of this block